docs/UserGuide.md

```diff
@@ -269,8 +269,7 @@ Format: `create-tag t/TAGNAME`
 * Tags are unique in WedLinker, there would not be any duplicated Tags.
 * Contacts can share Tags.
 * Tags are case-insensitive, so you cannot have both a 'hotel manager' and 'Hotel Manager' tag
-
-
+* If you want to create a Tag with a long name, it might not be shown clearly in the list. You can use [Tasks](#task-features) instead for better visuals.
```

Comment:
Is this supposed to be Tags not Tasks? The hyperlinks

Reply:
Also why are the users being directed to the features section, would it be better if we told them they could use `list-tasks` to have a better view of the tasks?

Reply:
Changed.